misc/websites_and_search_engines.talon

```diff
@@ -0,0 +1,5 @@
+open {user.website}: user.open_url(website)
```

Comment:
"open" is probably the right verb here, tho I have a slight hesitation having it at top-level, as there are various "open" commands in a number of contexts

Seems unlikely to run into ambiguity / misrecognitions, but figured I'd mention. 

Could potentially make it a bit longer, eg "open site", "launch site", "website", etc

But I don't feel super strongly on this one; "open" is fairly intuitive and short

Reply:
Yeah, I'm slightly hesitant on open too.